src/RunProcessAsTask.Tests/ProcessExTests.cs

```diff
@@ -1,151 +1,156 @@
 ﻿using System;
 using System.Diagnostics;
+using System.IO;
 using System.Linq;
+using System.Reflection;
+using System.Reflection.Internal;
 using System.Threading;
 using System.Threading.Tasks;
+using RunProcessAsTask;
 using Xunit;
 
 namespace RunProcessAsTask.Tests
 {
-    public static class ProcessExTests
-    {
-        public class RunAsyncTests
+        public static class ProcessExTests

```

Comment:
@MaxCheetham yeah, I'd have to agree with @EamonNerbonne - please undo the whitespace-only changes.  I realize tabs/spaces/tabwidth/etc is a holy war, but the "4 spaces, no tab" indent is what it's currently using and I'd like to keep it like that, at least for this PR. 😄